rules/macos/process_creation/proc_creation_macos_enable_root_account.yml

```diff
@@ -0,0 +1,41 @@
+title: Enable the Root Account
+id: 821bcf4d-46c7-4b87-bc57-9509d3ba7c11
+status: experimental
+description: Detects attempts to enable/add an account to the admin group, thus granting the root privilege
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1078.001/T1078.001.md
```

Comment:
```suggestion
```
Get only windows tests.

Reply:
Removed and added a correct reference! (Btw, the PR for T1078.003 ART is yet to be approved for other tests)